src/northbridge/amd/pi/agesawrapper.c

```diff
@@ -302,7 +302,7 @@ AGESA_STATUS agesawrapper_amdinitlate(void)
 	Status = AmdInitLate(AmdLateParams);
 	if (Status != AGESA_SUCCESS) {
 		agesawrapper_amdreadeventlog(AmdLateParams->StdHeader.HeapStatus);
-		ASSERT(Status == AGESA_SUCCESS);
+		printk(BIOS_WARNING, "AmdInitLate returned error!\n");
```

Comment:
@mek-x please drop it on mailing list for wider discussion. Maybe Kyosti or someone can help with that. Fix should be upstreamed. I think wider discussion is needed.